### PR TITLE
f-icons@2.10.2 - Change caret fill colour.

### PR DESCRIPTION
**Pull request by FayeCarter**
### Changed
- New Caret fill colour.

### CODE REVIEW

**Comment by ashleynolan** [comment]
Isn't this the same icon that's in the repo already? In terms of the fill colour – you should be able to change that using CSS anyway (that's what we do with other SVG's

**Review by DamianMullins — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v2.10.0
+------------------------------
+*November 13, 2020*
```
Comment (DamianMullins) [inline CHANGELOG.md:8]: Should November 23?

Comment (FayeCarter) [reply CHANGELOG.md:8]: It should have been yes 🤦

**Comment by DamianMullins** [comment]
@FayeCarter Did you mean to close this?

**Pull request closed without merging**